agent/command.go

```diff
@@ -26,13 +27,13 @@ func (a *Agent) runCommands(ctx context.Context, tc *taskContext, commands []mod
 		cmds, err = command.Render(commandInfo, tc.taskConfig.Project.Functions)
 		if err != nil {
 			tc.logger.Task().Errorf("Couldn't parse plugin command '%v': %v", commandInfo.Command, err)
-			if isTaskCommands {
+			if isTaskCommands || shouldSetupFail {
```

Comment:
to be honest, this should probably always fail (it's a parse error with the commands) even if it's in pre or somesuch.